scripts/setup-benchmarks/install_mongodb.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+mkdir -p /data/db
+
+# Start MongoDB server without sysctl
+mongod --dbpath /data/db --fork --logpath /var/log/mongodb.log
```

Comment:
this a great way to pre-load the data, but how it is setup putting the db into `/data/db` it looks as if it should be run inside a container but the README.md states it should be run outside
ideally this would be `$BENCHMARK_DIR/mongo_db` as in https://github.com/utcs-scea/KernMLOps/blob/0614a75224479808e21321eac337c3ad9151f578/scripts/setup-benchmarks/setup-gap.sh#L11

Reply:
Then it could be run inside or outside the container and work well, I would also recommend calling this load script from the setup stage of the benchmark but check if the dir exists first, if so skip the load step

Reply:
So just modify it to check for existence before creating it.

Reply:
Please change the `--dbpath` to be `--dbpath "$YCSB_BENCHMARK_DIR/mongo_db` and update the `mkdir -p` command above to create this new directory